test.js

```diff
@@ -0,0 +1,54 @@
+require( 'sugar' );
+var assert = require( 'assert' );
+var Promise = require( 'bluebird' );
```

Comment:
Package `bluebird` no longer exist as a dependencies so shouldn't be required :)